testing/tcc/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor:
+# Maintainer: Hinata Yanagi <hinasssan@gmail.com>
+pkgname=tcc
+pkgver=2017.10.19
+pkgrel=0
+pkgdesc="Tiny C Compiler"
+url="http://repo.or.cz/tinycc.git"
+arch="all"
+license="the GNU Lesser General Public License"
+depends="musl-dev"
+makedepends="gcc texinfo"
+commit=fc0188ffbcf29f856caedee8cef07d88be01e439
+source="http://repo.or.cz/tinycc.git/snapshot/${commit}.tar.gz"
+subpackages="${pkgname}-doc"
+
+builddir="$srcdir/"
```

Comment:
Please set `$builddir` to a meaningful value and use that value in `build()`, `check()` and `package()` instead of `"${builddir}tinycc-"*`. This is important because function like `default_prepare` rely on `$builddir`.